web/src/main/java/pl/edu/agh/security/AuthenticationService.java

```diff
@@ -0,0 +1,40 @@
+package pl.edu.agh.security;
+
+import java.util.Collections;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.stereotype.Service;
+import pl.edu.agh.model.UserRole;
+
+@Service
+public class AuthenticationService {
+
+    private static final String ROLE_PREFIX = "ROLE_";
```

Comment:
nie podoba mi sie ze ta stala jest tez jakby ukryta wewnatrz springa i jest wykorzystywana przy sprawdzaniu istnienia roli. Mamy 2 stale ktore trzeba pilnowac zeby sie nie rozlazly.
Jesli nie da sie dostac do tej springowej i jej wykorzystac to proponowalbym usuniecie calkowicie springowego prefiksu roli 
https://stackoverflow.com/questions/30146500/spring-security-3-2-7-httpservletrequest-isuserinrolestring-does-not-automatic#comment65293459_30155207
i explicite dolaczanie go zarowno tutaj jak i przy odczycie, albo w ogole z niego zrezygnowac

Reply:
racja, nie znalazłem opcji konfiguracji tego, poprawione :wink: